tools/screenshots-config.example

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Screenshot tool configuration file
+# In order to use the tool, copy this example to screenshots-config.sh
+# and fill the TOKEN_DEEPLINK field.
+
+# Auth token (required)
+TOKEN_DEEPLINK=
+
+# Uncomment the following APK key in order to override the default used apk
+# APK=../WordPress/build/outputs/apk/WordPress-wasabi-debug.apk
+
+# Uncomment the following PKG key in order to override the default used package config
+# Valid options are $PKG_DEBUG and $PKG_RELEASE
+# PKG=$PKG_DEBUG
+
+# Uncomment the following AVD key in order to override the default emulator
+#AVD=Nexus_5X_API_25_SCREENSHOTS
+
+# Uncomment the following RUN_DEV key in order to override the default device
+# Valid device are: PHONE, TAB7, TAB10 or ALL to run all of them
+#RUN_DEV=ALL
```

Comment:
For some reasons, the line endings of this file is CRLF and I see this when I source it from the main script:

```
screenshots-config.sh:2: command not found: ^M
screenshots-config.sh:6: command not found: ^M
screenshots-config.sh:9: command not found: ^M
screenshots-config.sh:12: command not found: ^M
screenshots-config.sh:16: command not found: ^M
screenshots-config.sh:19: command not found: ^M
```

Reply:
That's quite weird. I don't see these errors and I don't see the CR char checking with od. 
Any suggestion on this?